src/main/java/chess/service/ChessService.java

```diff
@@ -0,0 +1,87 @@
+package chess.service;
+
+import chess.dao.DatabaseChessDAO;
+import chess.domain.Board;
+import chess.domain.Rank;
+import chess.domain.Score;
+import chess.domain.Team;
+import chess.domain.piece.Piece;
+import chess.domain.position.Column;
+import chess.domain.position.Position;
+import chess.domain.position.Row;
+import chess.dto.PieceDTO;
+import chess.dto.RankDTO;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
+public class ChessService {
+
+    private final DatabaseChessDAO chessDAO;
```

Comment:
```suggestion
    private final ChessDAO chessDAO;
```

인터페이스가 아닌 구현체를 참조하고 있네요!
인터페이스를 사용해 참조하면 어떠한 장점이 있을지 고민해보아요!